core/modules/system/css/system.theme.css

```diff
@@ -585,6 +585,19 @@ tr:hover del.diffchange {
 }
 
 /**
+ * Hero blocks.
+ */
+ .hero-block {
+   background-size: cover;
+   background-repeat: no-repeat;
+   background-position: top center;
+   padding: 50px 100px;
+   text-align: center;
+   color: #ffffff;
```

Comment:
Scope creep. Color change can be added in contrib ;)

- -
Please excuse brevity and typos. Message typed with my thumbs from an Open
Source OS.

On Sep 10, 2016 9:55 AM, "wesruv" <notifications@github.com> wrote:

> In core/modules/system/css/system.theme.css
> <https://github.com/backdrop/backdrop/pull/1528#discussion_r78280196>:
>
> > @@ -585,6 +585,19 @@ tr:hover del.diffchange {
> >  }
> >
> >  /**
> > + * Hero blocks.
> > + */
> > + .hero-block {
> > +   background-size: cover;
> > +   background-repeat: no-repeat;
> > +   background-position: top center;
> > +   padding: 50px 100px;
> > +   text-align: center;
> > +   color: #ffffff;
>
> I take it back, the more I'm thinking about it the more I think it should
> be in config since it's dependent on the image uploaded and the person that
> uploads the image may not know CSS.
>
> Thinking the checkbox could be "Inverse text color", in case the default
> is light text on dark bg, or dark text on light bg?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/backdrop/backdrop/pull/1528/files/70450a66d386f11b3468e9351fff0078132d6ab6#r78280196>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAYSRwnl8BXYeJkceS9QEfj_QOH__Ielks5qouD8gaJpZM4J4xon>
> .
>